extensions/amp-access/0.1/amp-access.js

```diff
@@ -814,10 +817,11 @@ export class AccessService {
    *
    * @param {string} url
    * @param {string} eventLabel A label used for the analytics event for this action
+   * @param {boolean} noReturnUrl Don't perform a RETURN_URL replacement
    * @return {!Promise}
    */
-  loginWithUrl(url, eventLabel = '') {
-    return this.login_(url, eventLabel);
+  loginWithUrl(url, eventLabel = '', noReturnUrl) {
```

Comment:
You can't have non-default params after default. You have to move it up. Could you, though, explain why don't you need return URL?